src/algebraic_topology/dold_kan/faces.lean

```diff
@@ -0,0 +1,218 @@
+/-
+Copyright (c) 2022 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+
+import algebraic_topology.dold_kan.homotopies
+import data.nat.parity
+import tactic.ring_exp
+
+/-!
+
+# Study of face maps for the Dold-Kan correspondence
+
+TODO (@joelriou) continue adding the various files references below
```

Comment:
Just a general question: how closely is this series of PRs following a textbook source? To the extent it's possible to point the reader to written accounts that are close to the approach you're using, it will be very useful to provide these.

Reply:
The best I can do is what is already at the beginning of `homotopies.lean`. I do not know any source where the computations are done in very details and in a way that is similar to my implementation. (For example, in _Simplicial homopy theory_ by Goerss-Jardine from which I took some inspiration, there is a different convention for the definition of the normalized subcomplex: instead of the intersection of the kernels of the `d i` for `i>0` as you did, they took the kernel of all `d i` except for the biggest possible `i`. The `i>0` version is slightly better: there are less signs in the formulas...)

After the next 2 files are added (`projections.lean` and `p_infty.lean`), the normalized subcomplex shall be described as a direct factor of the alternating face map complex, and from there, the overall method of proof shall be similar to the strategies in the orignal papers.

Reply:
Okay, thanks!